core/store/migrations/migration1568390387/migrate.go

```diff
@@ -1,18 +1,33 @@
 package migration1568390387
 
 import (
+	"time"
+
 	"github.com/jinzhu/gorm"
 	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/assets"
 )
 
+type Encumbrance struct {
+	ID                     uint         `gorm:"primary_key;auto_increment"`
+	Payment                *assets.Link `gorm:"type:varchar(255)"`
+	Expiration             uint64
+	EndAt                  time.Time
+	Oracles                string  `gorm:"type:text"`
+	Aggregator             string  `gorm:"not null"`
+	AggInitiateJobSelector [4]byte `gorm:"not null"`
```

Comment:
How does this integrate with `sqlite`?

Reply:
`CREATE TABLE "encumbrances" ("id" integer primary key autoincrement,"payment" varchar(255),"expiration" bigint,"end_at" datetime,"oracles" text,"aggregator" varchar(255) NOT NULL,"agg_initiate_job_selector" blob NOT NULL,"agg_fulfill_selector" blob NOT NULL )`